test_manager/features/steps/tck_step_implementations.py

```diff
@@ -61,69 +48,125 @@ def create_sdk_data(context, sdk_name: str, command: str):
     context.action = command
 
 
-@then(u'the serialized uri received is "{expected_uri}"')
-def serialized_uri_received(context, expected_uri):
+@then(u'receives uri serialization "{expected_uri}"')
```

Comment:
@then(u'the serialized uri received is "{expected_uri}"') step is more clear

Reply:
The board agrees